source/extensions/filters/http/cache/http_cache.cc

```diff
@@ -0,0 +1,138 @@
+#include "extensions/filters/http/cache/http_cache.h"
+
+#include <algorithm>
+
+#include "common/protobuf/utility.h"
+
+#include "extensions/filters/http/cache/http_cache_utils.h"
+
+#include "absl/time/time.h"
+
+using absl::Duration;
+using absl::FromChrono;
+using absl::ZeroDuration;
+using Envoy::Extensions::HttpFilters::Cache::Internal::effectiveMaxAge;
```

Comment:
ditto here.  Even this one is just Internal::effectiveMaxAge used inside your namespace so it's really not too bad.

Reply:
Done.